bc_obps/reporting/models/report_unit.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from reporting.models.report_data_base_model import ReportDataBaseModel
+from reporting.models.report_source_type import ReportSourceType
+
+
+class ReportUnit(ReportDataBaseModel):
```

Comment:
When we talk about handling reports for multiple facilities, we probably want to consider whether all data models should have a fkey to that faciltiy_report for ease of flattening the data without needing a lot of transitive joins.

Reply:
We've added a fkey to the report version everywhere already, I wonder if that one would be useful too? I'll create a card for that

Reply:
https://github.com/bcgov/cas-reporting/issues/304